src/main/java/ch/njol/skript/command/ScriptCommand.java

```diff
@@ -145,6 +148,25 @@ public ScriptCommand(
 			this.permissionMessage = permissionMessage;
 		}
 
+		if (prefix != null) {
+			for (char c : prefix.toCharArray()) {
+				if (Character.isWhitespace(c)) {
+					Skript.warning("command /" + name + " has a whitespace in its prefix. Defaulting to '" + DEFAULT_PREFIX + "'.");
+					prefix = ScriptCommand.DEFAULT_PREFIX;
+					break;
+				}
+				// char 167 is §
+				if (c == 167) {
+					Skript.warning("command /" + name + " has an illegal character in its prefix. Defaulting to '" + DEFAULT_PREFIX + "'.");
```

Comment:
I think we should specify the exact character, otherwise the user will be questioning what the symbol is.

Reply:
It's impossible for that character to written in minecraft chat

Reply:
You can use Unicode `\u00A7` but that'll only be for console as the client will format the chat to be colour yes. You can also describe the symbol.

`You are not allowed to use the chat colour symbol in command...`

Reply:
People might think that `color symbol`is the ampersand. We could use the name of the symbol instead which is "section symbol", but I doubt a lot of people know what that is, although they can just google it.

Reply:
We need to explain it, I got confused.